Examples/ByFeature/FunctionalInjection/FunctionalInjection.fsproj

```diff
@@ -21,6 +21,8 @@
     <EmbeddedResource Include="ComputerHeating.feature" />
     <Compile Include="WalletSteps.fs" />
     <EmbeddedResource Include="Wallet.feature" />
+    <EmbeddedResource Include="ScenarioInformation.feature" />
```

Comment:
ScenarioContext? ScenarioMeta? Is there prior art for referring to this as `Information` ?

Reply:
I like the `ScenarioMetadata`.